cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -243,6 +248,19 @@ func (s *CMServer) Run(_ []string) error {
 	}
 	pvRecycler.Run()
 
+	var rootCAProvider func() []byte
+
+	if s.RootCAFile != "" {
+		rootCA, err := ioutil.ReadFile(s.RootCAFile)
+		if err != nil {
+			return fmt.Errorf("error reading root-ca-file: %v", err)
+		}
+		if _, err := x509.ParseCertificate(rootCA); err != nil {
```

Comment:
This assumes there is a single cert in the bundle... use `util/crypto#certsFromPEM` instead, which makes sure all the PEM blocks in the file decode correctly (fine to make that helper function public... we were just waiting for a need for it outside the package before doing so)